src/util/FactorUtil.js

```diff
@@ -111,6 +111,12 @@ function (Okta, TimeUtil) {
       description: '',
       iconClassName: 'mfa-okta-password',
       sortOrder: 14
+    },
+    'GENERIC_SAML': {
+      label: '',
+      description: 'factor.customFactor.description',
+      iconClassName: 'mfa-custom-factor',
+      sortOrder: 15
```

Comment:
can we confirm with Alex about the order of custom factor?

Reply:
confirmed, should be last